pkg/printers/internalversion/printers_test.go

```diff
@@ -47,2503 +45,3558 @@ import (
 	"k8s.io/kubernetes/pkg/apis/apps"
 	"k8s.io/kubernetes/pkg/apis/autoscaling"
 	"k8s.io/kubernetes/pkg/apis/batch"
+	"k8s.io/kubernetes/pkg/apis/certificates"
 	"k8s.io/kubernetes/pkg/apis/coordination"
 	api "k8s.io/kubernetes/pkg/apis/core"
 	"k8s.io/kubernetes/pkg/apis/discovery"
 	"k8s.io/kubernetes/pkg/apis/networking"
 	nodeapi "k8s.io/kubernetes/pkg/apis/node"
 	"k8s.io/kubernetes/pkg/apis/policy"
+	"k8s.io/kubernetes/pkg/apis/rbac"
 	"k8s.io/kubernetes/pkg/apis/scheduling"
 	"k8s.io/kubernetes/pkg/apis/storage"
 	"k8s.io/kubernetes/pkg/printers"
 	utilpointer "k8s.io/utils/pointer"
 )
 
-var testData = TestStruct{
-	TypeMeta:   metav1.TypeMeta{APIVersion: "foo/bar", Kind: "TestStruct"},
-	Key:        "testValue",
-	Map:        map[string]int{"TestSubkey": 1},
-	StringList: []string{"a", "b", "c"},
-	IntList:    []int{1, 2, 3},
+type TestPrintHandler struct {
+	numCalls int
 }
 
-type TestStruct struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Key               string         `json:"Key"`
-	Map               map[string]int `json:"Map"`
-	StringList        []string       `json:"StringList"`
-	IntList           []int          `json:"IntList"`
+func (t *TestPrintHandler) TableHandler(columnDefinitions []metav1beta1.TableColumnDefinition, printFunc interface{}) error {
+	t.numCalls++
+	return nil
 }
 
-func (in *TestStruct) DeepCopyObject() runtime.Object {
-	panic("never called")
+func (t *TestPrintHandler) getNumCalls() int {
+	return t.numCalls
 }
 
-func TestPrintUnstructuredObject(t *testing.T) {
```

Comment:
there's a lot of no-op code movement interleaved with rewrites that is making this difficult to review. would it be possible to split the no-op moves into an initial separate commit that separates out the tests we want to move elsewhere into their own file with no other changes?

For example, I *think* the following things moved without changes:
- TestPrintUnstructuredObject
- TestYAMLPrinter
- TestJSONPrinter
- TestFormatResourceName
- TestPrintHandlerError
- TestTemplatePanic
- TestNamePrinter
- TestTemplateStrings
- TestPrinters

Many other tests moved, then changed slightly to check cell output rather than string output. That's a great change, but it would be very helpful if those slight changes showed up in diffs, rather than as a diff of the entire function due to a move.